website/src/app/kb/deploy/resources/readme.mdx

```diff
@@ -23,13 +23,19 @@ To create a Resource, go to `Sites -> <site name> -> Add a Resource`.
 From there, you can select the type of Resource you want to create:
 
 - **DNS**: A domain name pattern to match.
-  - By default, the pattern will only match exactly the name you enter.
-  - To recursively match all subdomains, use a wildcard, such as
-    `*.example.com`. This will match `example.com`, `sub2.example.com`, and
-    `sub1.sub2.example.com`.
-  - To non-recursively match all subdomains, use a question mark, such as
-    `?.example.com`. This will match `example.com`, `sub1.example.com`, and
-    `sub2.example.com` **but not** `sub1.sub2.example.com`.
+  - By default, the pattern will only match the exact name you enter.
+  - To match all subdomains recursively, use a double-wildcard, such as
+    `**.example.com`. This will match `example.com`, `sub.example.com`, and
+    `sub.sub.example.com`.
+  - To match all subdomains non-recursively, use a single wildcard, such as
+    `*.example.com`. This will match `sub.example.com` but not
+    `sub.sub.example.com`.
+  - To match a single character, use a question mark, such as
+    `us-east?.example.com`. This will match `us-east1.example.com` but not
+    `us-eastXY.example.com`.
+  - Wildcards can be placed between domain components, e.g., `foo.*.example.com`
+    will match `foo.bar.example.com` or `foo.**.example.com` will match
+    `foo.bar.baz.example.com`.
```

Comment:
We probably want to merge this to staging and cut v1.2.0 clients before rolling this out to prod, so the time between merge and deploy might be a few days.

I'll add a todo item to uncomment this when we do the aug launch.

```suggestion
  - By default, the pattern will only match exactly the name you enter.
  - To recursively match all subdomains, use a wildcard, such as
    `*.example.com`. This will match `example.com`, `sub2.example.com`, and
    `sub1.sub2.example.com`.
  - To non-recursively match all subdomains, use a question mark, such as
    `?.example.com`. This will match `example.com`, `sub1.example.com`, and
    `sub2.example.com` **but not** `sub1.sub2.example.com`.  
{/*  - By default, the pattern will only match the exact name you enter. */}
{/*  - To match all subdomains recursively, use a double-wildcard, such as */}
{/*    `**.example.com`. This will match `example.com`, `sub.example.com`, and */}
{/*    `sub.sub.example.com`. */}
{/*  - To match all subdomains non-recursively, use a single wildcard, such as */}
{/*    `*.example.com`. This will match `sub.example.com` but not */}
{/*    `sub.sub.example.com`. */}
{/*  - To match a single character, use a question mark, such as */}
{/*    `us-east?.example.com`. This will match `us-east1.example.com` but not */}
{/*    `us-eastXY.example.com`. */}
{/*  - Wildcards can be placed between domain components, e.g., `foo.*.example.com` */}
{/*    will match `foo.bar.example.com` or `foo.**.example.com` will match */}
{/*    `foo.bar.baz.example.com`. */}
```